### PR TITLE
Prefer event target over element when calling setPointerCapture() (fixes #4)

### LINKED ISSUE #4: setPointerCapture() disables click events in Chrome
[This line](https://github.com/GoogleChromeLabs/pointer-tracker/blob/v2.0.3/lib/index.ts#L173) disables all click events inside the element, but only in Chrome.

I'm really sorry this is such a short ticket but I don't really have the time to create a reproducable case right now because my case is pretty involved, but I wanted to bring it up anyway. Please let me know if you need any additional information.

### PR DESCRIPTION
**Pull request by MidnightDesign**
See #4

### CODE REVIEW

**Comment by MidnightDesign** [comment]
@googlebot I signed it!

File: lib/index.ts [hunk 1]
```diff
@@ -170,7 +170,11 @@ export default class PointerTracker {
     // Add listeners for additional events.
     // The listeners may already exist, but no harm in adding them again.
     if (isPointerEvent(event)) {
-      this._element.setPointerCapture(event.pointerId);
+      const capturingElement = event.target ?? this._element;
+      if (!(capturingElement instanceof Element)) {
+        throw new Error('Expected the event target to be an Element.');
+      }
```
Comment (jakearchibald) [inline lib/index.ts]: How about?

```suggestion
      const capturingElement =
        event.target && 'setPointerCapture' in event.target
          ? event.target
          : this._element;
```

Avoids the need to throw an error.

Comment (MidnightDesign) [reply lib/index.ts]: Great idea

**Comment by jakearchibald** [comment]
Many thanks!

**Comment by jakearchibald** [comment]
I've published this as 2.1.0, along with a `stop()` method which removes listeners. You might be interested in using this in unmount callbacks.

**Comment by MidnightDesign** [comment]
@jakearchibald Already using it. Thanks!

**Pull request merged**